server/subscriptions.go

```diff
@@ -195,21 +195,25 @@ func (p *Plugin) GetSubscribedChannelsForRepository(pl webhookpayload.Payload) [
 	return subsToReturn
 }
 
-func (p *Plugin) Unsubscribe(channelID string, repo string) error {
+func (p *Plugin) Unsubscribe(channelID, repo string) (string, error) {
+	if len(strings.Split(repo, "/")) != 2 {
+		return requiredErrorMessage, nil
+	}
+
 	owner, repo := parseOwnerAndRepo(repo, p.getBaseURL())
 	if owner == "" && repo == "" {
-		return errors.New("invalid repository")
+		return "", errors.New("invalid repository")
```

Comment:
Would this work here as well?

```suggestion
		return requiredErrorMessage, nil
```